courses/admin.py

```diff
@@ -17,9 +17,10 @@ class CourseRunInline(admin.StackedInline):
 
 class CourseAdmin(admin.ModelAdmin):
     """ModelAdmin for Courses"""
-    list_display = ('title',)
+    list_display = ('position_in_program', 'title',)
     list_filter = ('program__live',)
     inlines = [CourseRunInline]
+    ordering = ('position_in_program',)
```

Comment:
I do not think we need it in the model: I needed here just because it's easier to see the courses